src/golang/lib/repos/integration.go

```diff
@@ -35,10 +36,10 @@ type integrationReader interface {
 	GetByServiceAndUser(ctx context.Context, service shared.Service, userID uuid.UUID, DB database.Database) ([]models.Integration, error)
 
 	// GetByUser returns the Integrations by the user with the ID userID in the organization with the ID orgID.
-	GetByUser(ctx context.Context, orgID string, userID uuid.UUID, DB database.Database) ([]models.Integration, error)
+	GetByUser(ctx context.Context, orgID string, userID utils.NullUUID, DB database.Database) ([]models.Integration, error)
```

Comment:
I think it makes more sense to leave this as `uuid.UUID`, since all the callers always pass in a userID in the current implementation. We should add a comment indicating that this method returns all integrations created by the org where the userID is equal to userID or it is NULL.